src/injectors/injector.ts

```diff
@@ -67,6 +68,29 @@ export abstract class InjectorBase implements Injector {
     }
 }
 
+export async function rewritePeriodKeybind() {
+    const configProvider = await ConfigProvider.create();
+    const config = configProvider.getConfig();
+
+    if (config.rewritePeriodKeybind) {
+        document.querySelectorAll('.js-github-dev-shortcut, .js-github-dev-new-tab-shortcut').forEach((elem) => {
+            const new_element = elem.cloneNode(true);
+            //@ts-ignore
+            elem.parentNode.replaceChild(new_element, elem);
+            new_element.addEventListener('click', (e) => {
+                //@ts-ignore
```

Comment:
Here we can cast `new_element` as an HTMLElement, so we won't need `//@ts-ignore`:
```suggestion
            const new_element = elem.cloneNode(true) as HTMLElement;
            elem.parentNode?.replaceChild(new_element, elem);
            new_element.addEventListener('click', (e) => {
```